src/components/script/navbar.js

```diff
@@ -4,7 +4,18 @@ class Navbar extends HTMLElement {
     super();
 
     // if you use `<core-navbar custom>`, this component does nothing
-    if (this.getAttribute("custom") !== null) return;
```

Comment:
The text is selectable in the button. you can make it unselectible by using ` user-select: none`. It is minor and you can merge without changing it.
![image](https://user-images.githubusercontent.com/79813230/192943803-f871a897-93f4-4016-ba65-2f71fc5a839a.png)

Reply:
resolved.